## Datasets
Datasets that available for download, mostly in NCBI SRA
* shendure lab
  * **plasma DNA, 60 samples, PE65 & PE100 read**
  * [Cell-free DNA Comprises an In Vivo Nucleosome Footprint that Informs Its Tissues-Of-Origin](doi: 10.1016/j.cell.2015.11.050)
  * [SRA](https://www.ncbi.nlm.nih.gov/Traces/study/?acc=PRJNA291063)
  * [Script](https://github.com/shendurelab/cfDNA)

## cfDNA
Scripts and documents for circulating cell-free DNA data analysis

## human REFERENCE genome
https://lh3.github.io/2017/11/13/which-human-reference-genome-to-use

## download much faster
ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/human_g1k_v37.fasta.gz

## Tutorial
https://qiubio.com/new/book/chapter-06/#%E7%AC%AC%E4%BA%94%E7%AB%A0-atac-seq%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90chapter-5-atac-seq-data-analysis

## Signal detection/statistics
1. Normalized coverage depth of each position
2. WPS (Windowed Protection Score) . [script]{https://github.com/shendurelab/cfDNA/blob/master/extractReadStartsFromBAM2Wig.py}
...

### peak calling software
https://github.com/taoliu/MACS
https://sites.google.com/site/danposdoc/

### ATAC-seq peak calling
https://yiweiniu.github.io/blog/2019/03/ATAC-seq-data-analysis-from-FASTQ-to-peaks/

## Chromatin Structure Research Methods (MNase DNase ATAC)
https://www.labome.com/method/Chromatin-Structure-Research-Methods.html
<img src="img/chromtain_access.jpg" width = "600" div align=left />

## ilustrate WHY there is a gap in the coverage of TF binding site
https://www.biostars.org/p/347703/
![image](img/TFBS_gap.png)


